src/PKSim.Core/Mappers/IndividualToIndividualBuildingBlockMapper.cs

```diff
@@ -80,8 +80,13 @@ bool shouldExportParameter(IParameter parameter)
             //Ontogeny factor are exported also in the expression profile
             if (parameter.IsExpressionOrOntogenyFactor())
                return false;
-            
-            return !_sameFormulaOrValueForAllSpeciesRepository.IsSameFormulaOrValue(parameter);
+
+            //different values for the same parameter in different species. It needs to be exported to the individual
+            if (!_sameFormulaOrValueForAllSpeciesRepository.IsSameFormulaOrValue(parameter))
+               return true;
+
+            //we only export if the value has changed from default
+            return parameter.ValueDiffersFromDefault();
```

Comment:
@Yuri05 @rwmcintosh 
this was the part missing...

Reply:
so easy and such a big impact...

Reply:
yeah. How did we miss that. I don't get this